docs/platform/howto/static-ip-addresses.rst

```diff
@@ -0,0 +1,108 @@
+Static IP addresses
+===================
+
+**Note: Each static IP addresses has an hourly price ranging from $0.025
+to $0.075 depending on region. The feature is recent and
+Aiven's**\ `pricing page <https://aiven.io/pricing>`__\ **doesn't
+include static IPs yet.**
+
+By default an Aiven service will use public IP addresses allocated from
+the cloud provider's shared pool of addresses for the cloud region. As a
+result, Aiven service nodes effectively get a random IP address. Because
+the addresses are shared with all other users of the cloud region, it's
```

Comment:
I find the part after "Because" hard to read, could you please try to rephrase it? Also please check that you're mentioning twice the granting access in the same sentence, maybe it's better to split it in two sentences?